gcp/modules/locust/swarm.tf

```diff
@@ -41,6 +41,31 @@ resource "null_resource" "locust_swarm_session" {
         RETRY_COUNT=$(($RETRY_COUNT+1))
       done
 
+      # Wait for target service to become rechable from within the locust pod before starting the test
+      RETRIES=100
+      echo "Waiting for ${var.locust_target_host} to become ready and reachable from all the locust pods..."
+      PODS=$(kubectl get pods --selector=component=worker -n locust -o=json --request-timeout=5s | jq -er '.items[].metadata.name')
+      for POD in $PODS; do
```

Comment:
Hard to read the indention levels so maybe I'm missing something, but I believe that if `kubectl get pods` fails, then `PODS` will be empty, everything in this loop will be skipped, and execution will continue (and fail).

Reply:
That is correct - my thinking here was that if the returned pods are empty (for whatever reason), the worst thing that happens is that we fail on the locust test itself. Also, it's consistent with error checking on the rest of the script, but you're right I can do more here.